src/features/types/responce.ts

```diff
@@ -0,0 +1,77 @@
+export interface fetchData {
+  data: data[];
+  page: number;
+  pageSize: number;
+  count: number;
+  totalCount: number;
+}
+export interface data {
+  id: string;
+  name: string;
+  supertype: string;
+  subtypes: string[];
+  hp: string;
+  types: string[];
+  evolvesFrom: string;
+  rules: string[];
+  attacks: attack[];
+  weaknesses: weakness[];
+  retreatCost: string[];
+  convertedRetreatCost: number;
+  set: {
+    id: string;
+    name: string;
+    series: string;
+    printedTotal: number;
+    total: number;
+    legalities: {
+      unlimited: string;
+      expanded: string;
+    };
+    ptcgoCode: string;
+    releaseDate: string;
+    updatedAt: string;
+    images: {
+      symbol: string;
+      logo: string;
+    };
+  };
+  number: string;
+  artist: string;
+  rarity: string;
+  nationalPokedexNumbers: number[];
+  legalities: {
+    unlimited: string;
+    expanded: string;
+  };
+  images: {
+    small: string;
+    large: string;
+  };
+  tcgplayer: {
+    url: string;
+    updatedAt: string;
+    prices: {
+      holofoil: {
+        low: number;
+        mid: number;
+        high: number;
+        market: number;
+        directLow: number;
+      };
+    };
+  };
+}
+
+type attack = {
```

Comment:
[the same about empty line between types and interface](https://github.com/mrrobot911/React-2023Q1/pull/1#discussion_r1375418545)